config/publisher-proxy/clusterrole.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Source: publisher-proxy/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: eventing-publisher-proxy-clusterrole
```

Comment:
(nit) `clusterrole` is redundant because it is already inferred by the `kind` property.